include/wx/qt/bitmap.h

```diff
@@ -105,6 +108,9 @@ class WXDLLIMPEXP_CORE wxMask : public wxMaskBase
     wxMask(const wxBitmap& bitmap);
     virtual ~wxMask();
 
+    // Construct a mask from QBitmap, takes ownership.
+    wxMask(QBitmap* qtBitmap);
```

Comment:
I think it should be
```suggestion
    explicit wxMask(QBitmap* qtBitmap);
```